src/groups/mqb/mqbnet/mqbnet_tcpsessionfactory.cpp

```diff
@@ -1082,8 +1081,7 @@ int TCPSessionFactory::start(bsl::ostream& errorDescription)
         bdlf::BindUtil::bind(&stopChannelFactory,
                              d_tcpChannelFactory_mp.get()));
 
-    bslmt::ThreadAttributes attributes =
-        bmqsys::ThreadUtil::defaultAttributes();
+    bslmt::ThreadAttributes attributes = bslmt::ThreadAttributes();
```

Comment:
```suggestion
    bslmt::ThreadAttributes attributes;
```
Possible to simplify